packages/fxa-auth-server/test/remote/totp_tests.js

```diff
@@ -18,514 +18,608 @@ const {
+        (response) => {
+          client = response;
+          assert.equal(
+            response.verificationMethod,
+            'totp-2fa',
+            'verification method set'
+          );
+          assert.equal(
+            response.verificationReason,
+            'login',
+            'verification reason set'
+          );
 
-  it('should request `totp-2fa` on login if user has verified totp token', () => {
-    return Client.login(config.publicUrl, email, password, {
-      ...testOptions,
-      keys: true,
-    }).then((response) => {
+          // Lets attempt to sign-in reusing session reauth
+          return client.reauth().then((response) => {
+            assert.equal(
+              response.verificationMethod,
+              'totp-2fa',
+              'verification method set'
+            );
+            assert.equal(
+              response.verificationReason,
+              'login',
+              'verification reason set'
+            );
+          });
+        }
+      );
+    });
+
+    it('should not create verified session after account reset with totp', async () => {
+      const newPassword = 'anotherPassword';
+
+      const client = await Client.login(config.publicUrl, email, password, {
+        ...testOptions,
+        keys: true,
+      });
       assert.equal(
-        response.verificationMethod,
+        client.verificationMethod,
         'totp-2fa',
         'verification method set'
       );
       assert.equal(
-        response.verificationReason,
+        client.verificationReason,
         'login',
         'verification reason set'
       );
+      await client.forgotPassword();
+      const code = await server.mailbox.waitForCode(email);
+      await client.verifyPasswordResetCode(code);
+      const res = await client.resetPassword(newPassword, {}, { keys: true });
+
+      assert.equal(
+        res.verificationMethod,
+        'totp-2fa',
+        'verificationMethod set'
+      );
+      assert.equal(res.verificationReason, 'login', 'verificationMethod set');
+      assert.equal(res.verified, false);
+      assert.ok(res.keyFetchToken);
+      assert.ok(res.sessionToken);
+      assert.ok(res.authAt);
     });
-  });
 
-  it('should not have `totp-2fa` verification if user has unverified totp token', () => {
-    return client
-      .deleteTotpToken()
-      .then(() => client.createTotpToken())
-      .then(() =>
-        Client.login(config.publicUrl, email, password, {
-          ...testOptions,
-          keys: true,
-        })
-      )
-      .then((response) => {
-        assert.notEqual(
-          response.verificationMethod,
-          'totp-2fa',
-          'verification method not set to `totp-2fa`'
-        );
-        assert.equal(
-          response.verificationReason,
-          'login',
-          'verification reason set to `login`'
-        );
-      });
-  });
+    it('should fail reset password without verifying totp', async () => {
```

Comment:
These are added tests, not sure why this file is getting reformatted

Reply:
I noticed some auto-formatting in a PR of mine also.  Maybe we upgraded a lint dep and now the defaults are different?  Kinda annoying.